src/python/pants/base/exiter.py

```diff
@@ -71,7 +72,7 @@ def exit(self, result=0, msg=None, out=None):
     :param out: The file descriptor to emit `msg` to. (Optional)
     """
     if msg:
-      print(msg, file=out or sys.stderr)
+      print(msg.encode('UTF8'), file=out or sys.stderr)
```

Comment:
`utf-8`?

Reply:
Either works, apparently. Although funny that I managed to spell this two ways in one review.